docs/guides/cli.md

```diff
@@ -61,6 +62,65 @@ Connection: keep-alive
 
 The body, headers, etc. for this response will be taken from the API description document.
 
+#### Request a specific example from a resource
+
+You can request a specific example from your document by using the `__example` query string paramenter.
+
+```bash
+curl -v http://127.0.0.1:4010/pets/123?__example=exampleKey
+```
+
+#### Change the dynamic/static flag for a particular request
+
+You can override the `dynamic` keyword (which decides whether the generated example is static or dynamic) through the `__dynamic` query string parameter.
+
+```bash
+curl -v http://127.0.0.1:4010/pets/123?__dynamic=false
+```
+
+## Proxy
+
+This command creates an HTTP server that will proxy all the requests to the specified upstream server. Prism will analyze the request coming in and the response coming back from the upstream server and report the discrepancies with what's declared in the provided OpenAPI document.
+
+**Note:** Currently the proxy command _expects_ a JSON Payload for both the request and the response. If the response's `content-type` header does not indicate a JSON-ish payload (`application/json`, `application/*+json`), then the payload will be just returned as text and send back to the original client.
+
+```
+prism proxy https://raw.githack.com/OAI/OpenAPI-Specification/master/examples/v2.0/yaml/petstore.yaml https://petstore.swagger.io/v2
+
+[CLI] …  awaiting  Starting Prism…
+[HTTP SERVER] ℹ  info      Server listening at http://127.0.0.1:4010
+[CLI] ●  note      GET        http://127.0.0.1:4010/pets
+[CLI] ●  note      POST       http://127.0.0.1:4010/pets
+[CLI] ●  note      GET        http://127.0.0.1:4010/pets/10
+```
+
+By default, the output violations will be reported on the standard output, but you can also specify to embed these in the HTTP Response:
+
+```bash
+prism proxy https://raw.githack.com/OAI/OpenAPI-Specification/master/examples/v2.0/yaml/petstore.yaml https://petstore.swagger.io/v2 --log httpHeader
```

Comment:
Did we stick with '--log httpHeader' we talked about some alternatives before, maybe the docs didn't get updated?

Reply:
We're still using `--log httpHeader`; I do not remember we agreed on something else but I might not recall correctly

Reply:
I don't think we agreed on an exact new set of options but we discussed some alternatives. I just listed a few ideas elsewhere in this PR we can try.